### PR TITLE
Perpetual layout menu

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.monday.com/boards/2218344956/pulses/2353751954

### CODE REVIEW

**Comment by ShynRou** [comment]
image
I changed a few things from the original design and the previous version:
1. The GSN stays on the right side as, left aligning it on the customize layout button makes no sense
2.  I didn't implement the merged dropdown, as it would need to be new custom component and I don't see the value in that atm. Could be something we should add to the Atomic Components Backlog.
3. I stretched the DataCard components vertically to fill the screen size, except for the Place New Order Card as this one requires a set height.

**Review by soulBit — CHANGES_REQUESTED** [review]
Code and functionality seems ok, but please clean up the build warnings before merging:
image

File: src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx
Comment (tiltom) [inline src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx:10]: Unnecessary, delete it pls.

File: src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx
Comment (tiltom) [inline src/app/pages/PerpetualPage/components/LayoutMenu/index.tsx:90]:
```suggestion
          className="tw-w-3 tw-ml-2"
```

**Review by tiltom — COMMENTED** [review]
I cannot request changes on my PRs so I submit these as comments :D

**Comment by ShynRou** [comment]
Fixed issues raised by @tiltom and removed dead variable `connected` @soulBit.
Critical dependency error is in a function that we do not use in the fronend tudor should fix that in the package.
transferAccountLocked should by addressed in #2071.

**Review by tiltom — COMMENTED** [review]
I cannot approve the request so take this as my approval :D

**Review by soulBit — APPROVED** [review]

**Pull request merged**
